components/eam/src/chemistry/utils/modal_aero_wateruptake.F90

```diff
@@ -23,7 +23,9 @@ module modal_aero_wateruptake
 
 public :: &
    modal_aero_wateruptake_init, &
-   modal_aero_wateruptake_dr
+   modal_aero_wateruptake_dr, &
+   modal_aero_kohler
```

Comment:
I can't find where this is used, why make it public?

Reply:
It's used for ECPP here:
src/physics/crm/ecpp/ecpp_modal_wetscav.F90

Reply:
BTW this is part of why ECPP was broken because there used to be a duplicate version of modal_aero_wateruptake.F90 that we deleted in PR #3738